src/lib/krad/t_packet.c

```diff
@@ -184,9 +186,13 @@ main(int argc, const char **argv)
 
     password = string2data("accept");
     noerror(make_packet(ctx, &username, &password, &packets[ACCEPT_PACKET]));
+    noerror(krad_packet_decode_request(ctx, "foo", &packets[ACCEPT_PACKET]->pkt,
```

Comment:
It seems we were missing calls to `krad_packet_decode_request()` in the tests. However, the structure definition of the `krb5_packet` generated by `krad_packet_new_request()` is private to `packet.c`. I had to move it to `internal.h` in order to access its `pkt` member.